examples/config/ca-config

```diff
@@ -1,3 +1,5 @@
 ca.node.profile=server
 ca.client.profile=client
-ca.endpoint=certificate-authority:8080
\ No newline at end of file
+ca.endpoint=certificate-authority:8080
+ca.serverCert.expiryHours=120
+ca.clientCert.expiryHours=120
```

Comment:
Do these look sensible?

Reply:
If it's not expensive, can we expire them more often?

Reply:
Shouldn't be that extensive, should it?

How does setting it to `12h` for start sound?

Reply:
> looks like it is a standard throught the UW

Probably got propagated through copy/pasta

I'd say start with 24h ?

Reply:
Sounds like a happy medium.

Reply:
> But I'm not sure whether we should spend this much of effort into it.
indeed